homeassistant/components/tuya/select.py

```diff
@@ -345,6 +345,24 @@
     # Dehumidifier
     # https://developer.tuya.com/en/docs/iot/categorycs?id=Kaiuz1vcz4dha
     "cs": (
+        SelectEntityDescription(
+            key=DPCode.MODE,
```

Comment:
This looks like a list with an limited set of options?  Should this be marked with the enum device class with providing these options instead?